### DISCUSSION TITLE
Compact table

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#9057] [Ideas]
It would be great if the table could be displayed even more compactly. Especially with a lot of data it would save a lot of space "downwards". Maybe with an option ->compact() ?

**Comment by sinnbeck** [comment]
Why not just collapse content?

https://filamentphp.com/docs/3.x/tables/layout#collapsible-content

**Comment by ghost** [comment]
Thanks for the idea. Unfortunately, the information must always be visible, so Collapse falls as a possibility out.

**Reply by sinnbeck** [reply]
There are several other ways of making the content more compact. 

https://filamentphp.com/docs/3.x/tables/layout#stacking-within-a-split

But I doubt filamnet can make a "fits all" solution as its hard to know where to split data.

**Comment by studiowizjo** [comment]
The elements within the table have classes prefixed with "fi-", allowing you to easily apply your own styles with reduced padding.

**Reply by ghost** [reply]
Oh, that's interesting information that I hadn't seen before. Do you have an example of how I can reduced padding?

**Reply by studiowizjo** [reply]
Maybe something like this:

```css
.fi-ta-cell > div {
    padding: 4px !important;
}
```

**Reply by ghost** [reply]
Thank you very much! That gave me the right hint. Implemented it as follows in a Custom CSS:

```
.fi-ta-cell, .fi-input, .fi-select-input, .fi-ta-cell div{
    padding-top: 0px !important;
    padding-bottom: 0px !important;
    margin-top: 0px !important;
    margin-bottom: 0px !important;
    border-radius: 0% !important;
    }

.flex{
    padding-top: 0px !important;
    padding-bottom: 0px !important;
    gap: 3px !important;

}
.fi-tabs{
    padding-top: 3px !important;
    padding-bottom: 3px !important;
}

.fi-header-heading{
    font-size: 1.5rem !important;
}
```

![image](https://github.com/filamentphp/filament/assets/63638852/9e840d46-54fc-44de-a7ba-7e16d9cc313b)
